llvm/tools/llvm-ir2vec/llvm-ir2vec.cpp

```diff
@@ -34,7 +42,7 @@
 #include "llvm/Support/raw_ostream.h"
 
 using namespace llvm;
-using namespace ir2vec;
+using namespace llvm::ir2vec;
```

Comment:
Instead of `using` statements, it might be better to wrap everything outside of `main` in an anonymous namespace inside the `llvm::ir2vec` namespace. I'm not sure what the coding standards are, but that's the pattern I see in other tools like `llvm-exegesis`.

Reply:
Done